processing/src/main/java/org/apache/druid/frame/key/RowKey.java

```diff
@@ -31,6 +31,7 @@
 public class RowKey
 {
   private static final RowKey EMPTY_KEY = new RowKey(new byte[0]);
+  private static final int OBJECT_OVERHEAD_SIZE_BYTES = 44; // Constant to account for hashcode and object overhead
```

Comment:
Can we add each item individually or mention in the comment how 44 was arrived at?